src/testdir/test_usercommands.vim

```diff
@@ -304,3 +304,135 @@ func Test_addr_all()
+        \           execute('command DoCmd'))
+  command! -addr=other DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       0    .     other                   :",
+        \           execute('command DoCmd'))
+
+  " Test with various -complete= argument values (non-exhaustive list)
+  command! -complete=arglist DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       0                    arglist       :",
+        \           execute('command DoCmd'))
+  command! -complete=augroup DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       0                    augroup       :",
+        \           execute('command DoCmd'))
+  command! -complete=custom,CustomComplete DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       0                    custom        :",
+        \           execute('command DoCmd'))
+  command! -complete=customlist,CustomComplete DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       0                    customlist    :",
+        \           execute('command DoCmd'))
+
+  " Test with various -narg= argument values.
+  command! -nargs=0 DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       0                                  :",
+        \           execute('command DoCmd'))
+  command! -nargs=1 DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       1                                  :",
+        \           execute('command DoCmd'))
+  command! -nargs=* DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       *                                  :",
+        \           execute('command DoCmd'))
+  command! -nargs=? DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       ?                                  :",
+        \           execute('command DoCmd'))
+  command! -nargs=+ DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n    DoCmd       +                                  :",
+        \           execute('command DoCmd'))
+
+  " Test with other arguments.
+  command! -bang DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
+        \        .. "\n!   DoCmd       0                                  :",
+        \           execute('command DoCmd'))
+  command! -bar DoCmd :
+  call assert_equal("\n    Name        Args       Address   Complete  Definition"
```

Comment:
The output of `:command DoCmd` with or without `-bar` option is identical.
Shouldn't `:command` outputs something to differentiate them?
as is the case with `-register`, `-bang` or `-buffer` which output
respectively `"`, `!` or `b`.